# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/x86_64-stat-fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Naur dietlibc_cvs-orig/include/sys/stat.h dietlibc_cvs/include/sys/stat.h
--- dietlibc_cvs-orig/include/sys/stat.h	2005-03-15 10:51:22.000000000 +0200
+++ dietlibc_cvs/include/sys/stat.h	2005-04-30 18:50:40.000000000 +0300
@@ -520,11 +520,11 @@
 	unsigned long	st_size;
 	unsigned long	st_blksize;
 	unsigned long	st_blocks;
-	unsigned long	st_atime;
+	signed long	st_atime;
 	unsigned long	__reserved0;
-	unsigned long	st_mtime;
+	signed long	st_mtime;
 	unsigned long	__reserved1;
-	unsigned long	st_ctime;
+	signed long	st_ctime;
 	unsigned long	__reserved2;
 	long		__unused[3];
 };
